cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -34,6 +34,19 @@ phases:
                 *) echo "Unsupported OS: $RELEASE" && exit {{ FailExitCode }} ;;
               esac
 
+      # Disable snap refresh to prevent background updates during build (Ubuntu only)
```

Comment:
[Non-blocking] We recently added the chnage to reduce the size of this template where one of the tricks we used was combine a few steps; lets do that going forward so thta we dont keep facing the template size issues in future.